packages/app/src/app/pages/Dashboard/Content/Sandboxes/index.tsx

```diff
@@ -6,9 +6,24 @@ import Filters from './Filters';
 
 import { Container, HeaderContainer, HeaderTitle } from '../elements';
 import { DashboardActions } from './Actions';
+import { ITemplate } from './types';
+
+interface Props {
+  sandboxes: any[];
+  Header: React.ComponentType;
+  SubHeader: React.ComponentType;
+  ExtraElement: React.ComponentType;
+
+  possibleTemplates: ITemplate[];
+  isLoading?: boolean;
+  hideOrder?: boolean;
+  hideFilters?: boolean;
+  page?: number;
+  actions?: any[];
```

Comment:
@CompuIves Shouldn't this be an array of `Action`s? 🤔

Reply:
I think they are different 🤔, its own structure. I should type it.